index.js

```diff
@@ -2538,6 +2668,7 @@
     PositiveNumber: PositiveNumber,
     RegExp: RegExp_,
     RegexFlags: RegexFlags,
+    Strict: Strict,
     StrMap: fromUncheckedUnaryType(StrMap),
     String: String_,
```

Comment:
I tried, but found that my editors built-in sorting mechanism changed the position of String.